manifests/07-downloads-helm.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: helm-download-links
+spec:
+  description: |
+    Helm helps you manage Kubernetes applications.
+    Helm Charts help you define, install, and upgrade even the most complex Kubernetes application.
```

Comment:
as discussed yesterday, this needs to indicate it is tech preview.

Reply:
Ah right!

Reply:
@siamaksade can you suggest a proper wording here, so that it is clear that we are talking about tech preview?

Reply:
@pedjak @sbose78 `Helm 3 is a package manager for Kubernetes applications which enables defining, installing and upgrading applications packaged as Helm Charts. Helm 3 is in Technology Preview feature on OpenShift 4`

Not sure if we can include links there but if we could, then _Technology Preview_ could be linked to: https://access.redhat.com/support/offerings/techpreview

Reply:
@siamaksade @sbose78 actually embedding html tags works, not sure if this is intended or not. 

I have updated the text, take a look how the text is now rendered.
![Screenshot-20191213001833-1340x141](https://user-images.githubusercontent.com/132690/70756908-22f6d100-1d3e-11ea-8fbd-320c5acc58f0.png)

Reply:
Evaluating the embedded link, I don't recall if that is intentional atm, but definitely need to look at this.

Reply:
Ok this text is rendered within `SyncMarkdownView`, we should be good here.
https://github.com/openshift/console/blob/ac67e8c3c66b73cea9f38cc0deb990b64c674259/frontend/public/components/command-line-tools.tsx#L25

Reply:
https://github.com/openshift/console/blob/88dc7f8343826e710e052ec559657648cd97a07a/frontend/public/components/markdown-view.tsx#L16
for reference, allowed HTML entities.